src/public/js/language/creator/pt.js

```diff
@@ -1,14 +1,14 @@
 var lang = new Object();
 
-lang.Loading = "";
-lang.Working = "Trabalhando...";
-lang.AYS = "Você tem certeza?";
-lang.Done = "Concluído!";
-lang.TOpen = "O template está abrindo...";
-lang.POpen = "A página está abrindo...";
-lang.TUpdate = "Salvando template...";
-lang.PUpdate = "Salvando página...";
-lang.PageOpened = "Página aberta!";
+lang.Loading = '';
```

Comment:
@acacio90  Could you use double quotes here please? and for every string where you used single quotes in this file, also change to double quotes.